### DISCUSSION TITLE
Encountering ERROR: native is not a valid partitioning type for pg_partman

### DISCUSSION THREAD

**Discussion by danqiye1** [pgpartman/pg_partman#691] [Q&A] [answered]
I am encountering the error stated in the title for the following query:
```sql
select
    public.create_parent(
        p_parent_table := 'my_shema.my_parent',
        p_control := 'timestamp',
        p_interval := '1 month',
        p_template_table := 'my_shema.my_parent_template',
        p_type := 'native'
    )
;
```
`pg_partman` extension is installed on the public schema. I am using pg_partman 5.1.0. Does anyone know what is wrong here?

**Comment by keithf4** [comment, marked as answer]
The `p_type` column has changed in version 5.x. Please review the release notes and current documentation

https://github.com/pgpartman/pg_partman/releases
https://github.com/pgpartman/pg_partman/blob/master/doc/pg_partman.md#create_parent
